### DISCUSSION TITLE
Grubość obramowania przy wybieraniu celu efektu

### DISCUSSION THREAD

**Discussion by itchystone** [emkarcinos/WMIAdventure#401] [Codding]
Wydaje mi się, że border zaznaczający wybrany cel efektu jest trochę zbyt cienki:

![image](https://user-images.githubusercontent.com/43492764/126209036-593a50d9-73fa-4b4e-b8af-a3a2455055d2.png)

Co powiecie na taki:

![image](https://user-images.githubusercontent.com/43492764/126208922-22490294-7307-4239-91bb-77af9bcb3b63.png)

**Comment by emkarcinos** [comment]
Hmm, ja nie jestem przekonany.

**Comment by mattyl006** [comment]
Mi nie przeszkadzał ten cienki, grubszy też mnie nie razi, jest mi to obojętne

**Comment by itchystone** [comment]
Spoko, to niech zostanie tak jak jest
